importer/balanced/builder.go

```diff
@@ -27,27 +27,29 @@
 //     should be used (like the trickle builder does).
 //     (See https://github.com/ipfs/go-ipfs/pull/5120.)
 //
-//     +-------------+
-//     |   Root 4    |
-//     +-------------+
-//     |
-//     +--------------------------+----------------------------+
-//     |                                                       |
-//     +-------------+                                         +-------------+
-//     |   Node 2    |                                         |   Node 5    |
-//     +-------------+                                         +-------------+
-//     |                                                       |
-//     +-------------+-------------+                           +-------------+
-//     |                           |                           |
-//     +-------------+             +-------------+             +-------------+
-//     |   Node 1    |             |   Node 3    |             |   Node 6    |
-//     +-------------+             +-------------+             +-------------+
-//     |                           |                           |
-//     +------+------+             +------+------+             +------+
-//     |             |             |             |             |
-//     +=========+   +=========+   +=========+   +=========+   +=========+
-//     | Chunk 1 |   | Chunk 2 |   | Chunk 3 |   | Chunk 4 |   | Chunk 5 |
-//     +=========+   +=========+   +=========+   +=========+   +=========+
+// Example balanced DAG:
```

Comment:
A line of text appears to be required here. Otherwise, `gofmt` sees the diagram as part of the numbered list above, and indents and reformats it.